frappe_io/www/accounting/docs/payment.md

```diff
@@ -0,0 +1,25 @@
+<!-- add-next-prev-links -->
+## Payments
+
+In order to capture a Payment from a Customer, you create a Payment entry. You can create a payment only for a submitted invoice.
+
+To create a new payment, follow the below steps:
+
+##### Payment Option :
+- For a submitted Invoice, click on the Actions drop-down, you will see a payment option.
+
+<img  src="/accounting/assets/img/paymentoption.png"
+      class="screenshot"
+/>
+
+##### Payment Modal :
+- By clicking on the payment option, a payment modal will appear in which most of the details will already be fetched from the invoice.
+
+<img  src="/accounting/assets/img/paymentmodal.png"
+      class="screenshot"
+/>
+
+##### Payment Submission :
+- By clicking on the save button, payment will be completed.
```

Comment:
Add this: "You can check the corresponding ledger entries by clicking on **Ledger Entries** in the actions dropdown"